### ISSUE TITLE
Details about Clustering to get Pose Centroids?

### ISSUE THREAD

**Issue by ashar-ali** [AVAuco/ddp#3]
Hi @mjmarin , 

Do you have any plans to release the training code soon?
Specifically, I would like to understand more about how you clustered the K centroid poses from training data in the first place?

**Comment by mjmarin** [comment]
Hi @ashar-ali ,
sorry for the late reply, but I'm a bit overloaded.
We used k-means for clustering. In particular, we used VLFeat implementation:
`[centroids, labels, foo] = vl_kmeans(images.joints, numClusters, 'Initialization', 'plusplus', 'NumRepetitions',3, 'verbose');`
I hope that this helps.
